tests/publiccloud/storage_perf.pm

```diff
@@ -202,7 +203,9 @@ sub run {
             my $db = get_var('PUBLIC_CLOUD_PERF_DB', 'perf_2');
             my $token = get_required_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
             my $org = get_var('PUBLIC_CLOUD_PERF_DB_ORG', 'qec');
-            influxdb_push_data($url, $db, $org, $token, $data) if (check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
+            $res = influxdb_push_data($url, $db, $org, $token, $data, proceed_on_failure => 1, timeout => $db_timeout)
+              if (check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
+            return unless ($res);
```

Comment:
I believe this should be under a `(check_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1))` block, because $res is assigned on condition and then we return based on value on variable.

Reply:
No , I did proper tests for this syntax in a local environment of mine, I cheched the results (if you want I can provide the code and results) and indeed it works well, as also the CI pass and VR tests proof. 
The only attention needed is to _avoid_ in-line declaration like `my $res = ... if ...`, causing even CI to fails: infact I declared before that $res.

Reply:
Ah, moreover, the `undef` case on unmatching condition is also wanted, used in the line 208.

Reply:
A block is more readable than relying on implicit assumptions, but anyway.

Reply:
Generally speaking, you are right, but also I preferred a minimalistic form reducing my changes, keeping the original structure with only a newline added, other than the assignment.